src/components/TopSection/Filter/Adult.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import PropTypes from 'prop-types';
+import { decrementAdult, incrementAdult } from '../../../actionCreator';
+
+const Adult = ({ labelText, endNumber }) => {
+  const adult = useSelector((state) => state.adult);
+  const dispatch = useDispatch();
+
+  const decrementAdultHandler = (e) => {
+    e.preventDefault();
+    dispatch(decrementAdult());
+  };
+
+  const incrementAdultHandler = (e) => {
+    e.preventDefault();
+    dispatch(incrementAdult());
+  };
+
+  return (
+    <>
+      <div className="item">
+        <span>{labelText}</span>
+        <div className="choose">
+          <button
+            onClick={decrementAdultHandler}
+            className={adult === 0 ? 'disabled-btn btn' : 'btn'}
```

Comment:
read about classnames lib here https://www.npmjs.com/package/classnames (it's as an advice only)